components/python/pygobject-3-legacy/pygobject-3-27.p5m

```diff
@@ -0,0 +1,70 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Gary Mills
+# Copyright 2016 Alexander Pyhalov
+#
+
+# This file intentionally hardcodes Python 2.7 instead of using the PYVER
+# subsititutions to avoid triggering the auto-generation of a generic
+# library/python/pygobject-3 in this component, since that's being done in
+# ../pygobject-3 instead to cover the other python 3.x versions correctly.
+
+set name=pkg.fmri value=pkg:/library/python/pygobject-3-27@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Python bindings for GObject"
+set name=info.classification value="org.opensolaris.category.2008:Development/Python"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license="LGPL v2.1"
+
+depend type=require fmri=runtime/python-27
+depend type=conditional fmri=library/python/pygobject-3 \
+    predicate=runtime/python-27
+depend type=require fmri=library/desktop/gtk3
+
+link path=usr/include/pygobject-3.0/pygobject.h target=pygobject.h-2.7 \
+	mediator=python3 mediator-version=2.7
+link path=usr/lib/$(MACH64)/pkgconfig/pygobject-3.0.pc target=pygobject-3.0.pc-2.7 \
+	mediator=python3 mediator-version=2.7
+
```

Comment:
You are making python-2.7 a python-3 version in OpenIndiana with these two links and its mediator settings.
We don't actually seem to have a package that depends on pygobject-3-27 and retaining this kind of compability seems to be not worth the problems this will probably create.

Reply:
I wanted to use an existing mediator and also realized that the "python3" mediator could never assume the value "2.7".  Thus, the symlinks would never be created.  The only alternative left now is to delete those lines.  This change means that the package would still be published, but that dependent packages could not be built.  I'll delete those lines in the next push.